<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DataGridViewDataErrorEventArgs.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac507424dd07417db23cdbec88dd22f5097f25643bc.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07424dd07417db23cdbec88dd22f5097f25643bc</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Provides data for the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> event.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> イベントのデータを提供します。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Handling the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event lets you handle exceptions thrown by code outside your control (for example, by an external data source).</source>
          <target state="translated">処理、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph>イベントできます (たとえば、外部データ ソース) をコントロールの外部コードによってスローされる例外を処理します。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property to determine the state of the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> at the time of the exception.</source>
          <target state="translated">使用して、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph>プロパティの状態を判断、<ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph>例外時にします。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property to retrieve the exception data.</source>
          <target state="translated">使用して、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph>例外データを取得するプロパティです。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>If you want to handle the exception by additional event handlers, set the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph2">`true`</ph>.</source>
          <target state="translated">追加のイベント ハンドラーによって例外を処理する場合は、設定、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph>プロパティを<ph id="ph2">`true`</ph>です。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> and <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> properties normally indicate the cell in which the data error occurred.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph>と<ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph>プロパティは、通常、データ エラーが発生したセルを示します。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>When the error occurs in an external data source, however, the data source may not provide the column in which the error occurred.</source>
          <target state="translated">外部データ ソースでエラーが発生するときにただし、データ ソースでは、エラーが発生した列が提供されません可能性があります。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>In this case, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> property typically indicates the column of the current cell at the time of the error.</source>
          <target state="translated">ここで、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph>プロパティは通常、エラーの発生時に現在のセルの列を示します。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The following code example demonstrates how to respond to information provided by the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> class.</source>
          <target state="translated">次のコード例は、によって提供される情報に応答する方法を示します、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph>クラスです。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">この例で使用できる例の一部である、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph>クラスの概要に関するトピック。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The exception that occurred.</source>
          <target state="translated">発生した例外。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The column index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> が発生したセルの列インデックス。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The row index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> が発生したセルの行インデックス。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>A bitwise combination of <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values indicating the context in which the error occurred.</source>
          <target state="translated">エラーが発生したコンテキストを示す <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> 値のビットごとの組み合わせ。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Gets details about the state of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> when the error occurred.</source>
          <target state="translated">エラーが発生したときの <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> の状態の詳細を取得します。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>A bitwise combination of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values that specifies the context in which the error occurred.</source>
          <target state="translated">エラーが発生したコンテキストを指定する <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> 値のビットごとの組み合わせ。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Members of the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumeration may be combined using the bitwise <ph id="ph2">`OR`</ph> operator to represent the state of a data-bound <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> when a data error occurred.</source>
          <target state="translated">メンバー、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph>列挙体は、演算を使用して組み合わせることができます<ph id="ph2">`OR`</ph>データ バインドの状態を表す演算子<ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph>データ エラーが発生しました。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>For example, if a user enters an invalid cell value (such as entering a name in a cell that requires a date) and then selects a different cell, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will try to commit the invalid cell value.</source>
          <target state="translated">たとえば、ユーザー (日付を必要とするセルの名前を入力します) など、無効なセル値を入力し、別のセルを選択、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph>は無効なセルの値をコミットを再試行してください。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>When the commit fails, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will raise a <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event whose <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property will have a value of <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> and <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</source>
          <target state="translated">コミットが失敗したときに、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph>が生成されます、<ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph>イベントが<ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph>の値を持つ<ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph>と<ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>です。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>The following code example demonstrates how to investigate the error context.</source>
          <target state="translated">次のコード例では、エラーのコンテキストを調査する方法を示します。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">この例で使用できる例の一部である、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph>クラスの概要に関するトピック。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>Gets the exception that represents the error.</source>
          <target state="translated">エラーを表す例外を取得します。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error.</source>
          <target state="translated">エラーを表す <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>To handle the exception by more than one <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, throw the exception again at the end of a handler by setting the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph3">`true`</ph>.</source>
          <target state="translated">1 つ以上の例外を処理する<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>を設定して、ハンドラーの最後にもう一度例外をスローする、<ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph>プロパティを<ph id="ph3">`true`</ph>です。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>The following code example demonstrates checking the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property.</source>
          <target state="translated">次のコード例では、チェックを示しています、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph>プロパティです。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">この例で使用できる例の一部である、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph>クラスの概要に関するトピック。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Gets or sets a value indicating whether to throw the exception after the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> delegate is finished with it.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> デリゲートが終了した後に例外をスローするかどうかを示す値を取得または設定します。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the exception should be thrown; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated">例外をスローする場合は <ph id="ph1">&lt;see langword="true" /&gt;</ph>。それ以外の場合は <ph id="ph2">&lt;see langword="false" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The default is <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated">既定値は、<ph id="ph1">&lt;see langword="false" /&gt;</ph> です。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Set this property to <ph id="ph1">`false`</ph> if the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> has dealt with the exception represented by the <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property, and you do not want to propagate the exception to another <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> that may handle the <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event.</source>
          <target state="translated">このプロパティを設定<ph id="ph1">`false`</ph>場合、<ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>によって表される例外に取り組んでおり、<ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph>プロパティ、およびする、別の例外を反映しないように<ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>を処理することがあります、<ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph>イベント。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>For debugging purposes, it can be useful to propagate the exception and inspect the stack trace for information regarding the context of the error.</source>
          <target state="translated">デバッグのため、例外が反映され、スタック トレース、エラーのコンテキストに関する情報を検査すると便利ですができます。</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The following code example demonstrates using the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to indicate that the exception should not be thrown.</source>
          <target state="translated">次のコード例では、使用方法を示します、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph>例外がスローされないことを示すプロパティです。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">この例で使用できる例の一部である、<ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph>クラスの概要に関するトピック。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>When setting this property to <ph id="ph1">&lt;see langword="true" /&gt;</ph>, the <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> property value is <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">このプロパティを <ph id="ph1">&lt;see langword="true" /&gt;</ph> に設定した場合、<ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> プロパティの値は <ph id="ph3">&lt;see langword="null" /&gt;</ph> になります。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>